documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -349,17 +349,16 @@ The {scope} is the specific scope defined to protect this operation.
 
 ### Mandatory template for `info.description` in CAMARA API specs
 
-The documentation template below must be used as part of the API documentation in  `info.description` property in the CAMARA API specs:
-
+The documentation template below must be used as part of the API documentation in `info.description` property in the CAMARA API specs:
 
 ```
 ### Authorization and authentication
 
-The "Camara Security and Interoperability Profile" provides details on how a client requests an access token. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the released version of the Profile.
+The "Camara Security and Interoperability Profile" provides details on how a client requests an access token.
```

Comment:
replace "client" by "API consumer"

Can this text be more explicit and say "how an API consumer SHALL request an access token" ?

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/214/commits/00bac931568697c72ba5cbfcfea76e83fffc8329. I've restored the link to the ICM WG, which I missed. I prefer to keep it as well. It was a change originally requested in #182.